os/drivers/Makefile

```diff
@@ -88,6 +88,7 @@ include lwnl${DELIM}Make.defs
 include net$(DELIM)Make.defs
```

Comment:
Please change commit description pm lock unlock to suspend resume

Reply:
updated